### DISCUSSION TITLE
Monitor historical bandwidth/traffic usage from/to internet per LAN IP ?

### DISCUSSION THREAD

**Discussion by jult** [akvorado/akvorado#1358] [Q&A]
So, I'm just wondering, before I go about installing this; I run a debian 12 router, it has 1 interface on the WAN side, does NAT to a bridge of LAN interfaces (9 in total), with about 60 devices in the LAN, all connected to the internet through this gateway/router.
Can I have akvorado create a list of traffic usage totals per time-period (day/week/month) per LAN IP, excluding local/LAN traffic? So only showing bytes to/from the WAN side (which is NAT to a CGNAT modem bridged IP).

**Comment by vincentbernat** [comment]
Yes, it should work. You can make queries that filter on interface names or descriptions.

**Comment by jult** [comment]
Also when this is all installed on the same device, i.e. this debian router? Especially the NAT issue seems to be problematic for some tools,

**Comment by knewtbork** [comment]
Sorry to dredge up an old topic, but I've been checking out the demo site and had the same question. Say for example I wanted to know how many gigabytes of data went out an interface last month.  All the filters seem to give results as per-second rather than total volume of data.
